plugin/tagbar.vim

```diff
@@ -86,7 +86,7 @@ call s:setup_options()
 
 if !exists('g:tagbar_iconchars')
     if has('multi_byte') && has('unix') && &encoding ==# 'utf-8' &&
-     \ (empty(&termencoding) || &termencoding ==# 'utf-8')
+     \ (!has('termencoding') || empty(&termencoding) || &termencoding ==# 'utf-8')
```

Comment:
I tested this on by Neovim bus it does not identify when the option does exist. This is not a feature flag (what `has()` usually tests for), it is an option. According to the docs I found `exists()` is the way to check this, with either `&...` for just whether the option exists or `+...` for whether it's actually a functional option (as opposed to legacy deprecated or whatever). Can you check the other PR and make sure it works for you?